action.yml

```
@@ -42,9 +42,13 @@ runs:
         # in order to increase the limit of 60 requests per hour per IP address to a higher value that's also counted
         # per GitHub account.
         # Caching is disabled in order not to receive stale responses from Varnish cache fronting GitHub API.
+        RELEASE_HEADER=''
+        if [ ! -z "${{github.token}}" ]; then
+          RELEASE_HEADER="--header 'authorization: Bearer ${{ github.token }}'"
```

```suggestion
          RELEASE_HEADER=("--header" "'authorization: Bearer ${{ github.token }}'")
```
We need to pass two args as in other way it will break
See: https://stackoverflow.com/a/7454624/1387612